.github/workflows/ci.yml

```
@@ -71,7 +71,10 @@ jobs:
           python -m pytest test -v --cov=./uxarray --cov-report=xml
 
       - name: Upload code coverage to Codecov
+        if: github.repository == 'NCAR/geocat-comp'
```

I believe this should be the following? 

```suggestion
        if: github.repository == 'UXARRAY/uxarray'
```

---

Oh, a good proof of one should avoid copy-paste late in the evening :) Thanks for catching!